### PR TITLE
Repoint portrait table and transplant vanilla table.

### PR DESCRIPTION
**Pull request by Crazycolorz5**
So y'know how we have a bajillion portraits and not enough table space for them...

### CODE REVIEW

**Comment by Darrman** [comment]
For whatever reason, some class cards crash the game when viewed as of this commit. Soldiers, mercenaries and pirates are examples of this in the first two chapters. It's possible I might have broken something adding new portraits in with the repoint. Have the class cards been moved from their vanilla position?

**Comment by Crazycolorz5** [comment]
> 
> 
> For whatever reason, some class cards crash the game when viewed as of this commit. Soldiers, mercenaries and pirates are examples of this in the first two chapters. It's possible I might have broken something adding new portraits in with the repoint. Have the class cards been moved from their vanilla position?

The class cards are the same as they are in vanilla since i just transplanted the vanilla table. It's likely that some of the cards got overwritten by new portrait IDs? If reverting those doesn't work I can do a more detailed check at some point, lmk.

**Comment by Darrman** [comment]
> The class cards are the same as they are in vanilla since i just transplanted the vanilla table. It's likely that some of the cards got overwritten by new portrait IDs? If reverting those doesn't work I can do a more detailed check at some point, lmk.
I reverted them in a separate branch. The game still crashes. I've made some observations on what crashes the game and what doesn't, though. This is all just me hypothesizing, though.

- The first 0x80 portraits were repointed.
- Class cards below 0x80 do not crash the game, as they were included in this repoint.
- Newly added portraits above 0x80 work without issue.
- However, portraits that were already above 0x80 can no longer be found, causing the game to crash.
- This includes many class cards and the supply menu, as the wagon icon on that screen is a minimug, and this minimug's portrait is found above 0x80.
In short, I have a feeling that leaving parts of the portrait table unrepointed is causing issues. The vanilla game has 0xAC valid portraits, but only 0x80 of these were copied. While adding a portrait above 0x80 is fine, as the pointers are correct, the rest of the portraits are probably using their original pointers and are causing issues. Copying the entire table when repointing might fix the issues.

**Comment by Crazycolorz5** [comment]
> However, portraits that were already above 0x80 can no longer be found, causing the game to crash.

I wasn't aware that there were more than 0x80 in vanilla. Should be easily fixable. I'll get it tonight.
